### DISCUSSION TITLE
Display doesn't work (also)

### DISCUSSION THREAD

**Discussion by SR7673** [tbnobody/OpenDTU#884] [Q&A] [answered]
Hi all, 
yes, i read the other threads regarding not working dsplays but this didn't solve my problem.
I'm using a SSD1306 display (https://aliexpress.com/item/32683094040.html) connected to 5V.
Connected like here: https://blog.helmutkarger.de/wp-content/uploads/ESP32mitDisplay.png
The DTU works fine but the display does not. I've flashed v23.4.28 but nothing else. Is it necessary to flash something for running the dsplay? I found no documentation for that.
In the hardware settings i found for the display:
clk 255
cs 255
data 255
reset 255
type 0
Looks like it is unconfigured to me.
So what do i need to do to get it running?
Thanks in advance.

**Comment by tbnobody** [comment, marked as answer]
Did you follow the instructions in the documentation? https://github.com/tbnobody/OpenDTU/blob/master/docs/Display.md

**Comment by SR7673** [comment]
Just excellent! One of the best projects ever on github!
